site/en/docs/privacy-sandbox/videos/index.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox video library'
+subhead: >
+  Find overviews, walkthroughs, and conceptual discussions in these Privacy Sandbox videos.
+description: >
+  Privacy Sandbox video library
+date: 2023-06-26
+authors:
+  - nmichell
+---
+
+If you're looking for Privacy Sandbox resources, check out the videos below. Some are recorded Privacy Sandbox office hours. You can find upcoming office hours on the [events page](/docs/privacy-sandbox/events/).
+
+{% Aside %}
+Keep in mind that while the concepts discussed in these videos are still relevant, some technical details and code will have changed.
+
+Refer to the linked documentation for the latest details.
+{% endAside %}
+
+## What is the Privacy Sandbox?
+
+{% Columns %}
+{% Column %}
+<iframe width="280" src="https://www.youtube.com/embed/NKz5oT6kXI4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Preferable to use the specific YouTube embed, I think.
https://developer.chrome.com/docs/handbook/how-to/add-a-youtube-embed/